.github/coderabbit.yaml

```
@@ -0,0 +1,266 @@
+# 🤖 CodeRabbit Configuration for MARIA Platform
+# Specialized AI code review settings for the MARIA development platform
+
+# General Review Settings
+reviews:
+  # Enable comprehensive review
+  enable: true
+  # Review all file types relevant to MARIA
+  file_types:
+    - "**/*.ts"
+    - "**/*.tsx" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)